src/components/AccountSummaryCard.tsx

```diff
@@ -29,7 +29,11 @@ const AccountSummaryCard = ({ className }: { className?: string }) => {
 
   const totalBalance = addresses.reduce((acc, address) => acc + BigInt(address.details.balance), BigInt(0))
   const totalAvailableBalance = addresses.reduce((acc, address) => acc + address.availableBalance, BigInt(0))
-  const totalLockedBalance = addresses.reduce((acc, address) => acc + BigInt(address.details.lockedBalance), BigInt(0))
+  const totalLockedBalance = addresses.reduce((acc: bigint | undefined, { details: { lockedBalance } }) => {
+    if (acc === undefined) return acc
+    if (lockedBalance === undefined) return undefined
+    return acc + BigInt(lockedBalance)
+  }, BigInt(0))
```

Comment:
I disagree with the "regardless this issue should still be fixed in the wallet UI". We don't have to make the wallet backwards compatible every time the API changes.